src/geometry.F90

```diff
@@ -475,6 +475,68 @@ subroutine cross_surface(p, last_cell)
              &// trim(to_str(surf%id)))
       end if
       return
+    elseif (surf % bc == BC_PERIODIC .and. run_mode /= MODE_PLOTTING) then
+      ! =======================================================================
+      ! PERIODIC BOUNDARY
+
+      ! Do not handle periodic boundary conditions on lower universes
+      if (p % n_coord /= 1) then
+        call handle_lost_particle(p, "Cannot period particle " &
+             // trim(to_str(p % id)) // " off surface in a lower universe.")
```

Comment:
This sounds weird.  Maybe "Cannot transfer particle N across periodic surface in a lower universe."

To be really clear, maybe we should also add "Boundary conditions must be applied to universe 0"